backend/models/languages.py

```
@@ -0,0 +1,12 @@
+"""
+Language configuration models.
+"""
+from pydantic import BaseModel
```

Ruff (0.11.9)

`pydantic.BaseModel` imported but unused

Remove unused import: `pydantic.BaseModel`

(F401)